
02032022
\section{\emph{Escherichia Coli} genomics}
\emph{Escherichia Coli} is Gram-negative, facultative anaerobic, rodshaped, coliform bacterium, it pertains to the phylum of proteobacteria and to the family of Enterobacteriaceae. It can be grown easily and inexpensively. It has got genome with a length between 4.5 - 4.7 M bases, that includes 4000 to 5000 genes. it has about 4000-5000 genes, and about seven ribosomal RNA operons. Only the 38\% of the genes of K-12 (one of the most studied bacterial strains of \emph{E. coli}) were experimentally identified, overall 40-50\% of the genes are to date without a known function.
The original \emph{E. coli} strain K-12 was obtained from a stool sample of a
diphtheria patient in Palo Alto, CA in 1922.

\subsection{\emph{E. coli} long-term evolution experiment}
The \emph{E. coli} long-term evolution experiment led by Richard Lenski is one of the longest evolutionary experiments ever made (/href{youtube.com/watch?v=w4sLAQvEH-M}{The longest evolutionary experiment ever made}). The experiment started on 24th February 1988, and since that moment 12 populations of \emph{E. coli} have been cultivated in the same environment. After each day (corresponding to the time of development of approximately 7 generations), a portion of bacteria from each flask was introduced in a new one, and let proliferating in it. Every 500 generation, it has been saved a sample of the bacteria of each flask, in order to track the evolutionary changes made. Today the experiment is on-going, and researchers reached approximately the 66000th generation. The study suggests a series of conclusions, to cyte "long-term adaptation to a fixed environment can be characterized by a rich and dynamic set of population genetic processes, in stark contrast to the evolutionary desert expected near a fitness optimum" (Good et al 2017). In fact, despite of the fixed environment, some bacteria developed the capacity to aerobically grow on citrate, which is unusual in \emph{E. coli} (around generation 31,000) and developed complex mutation patterns.


\subsection{\emph{E. coli} strains}
\emph{E. coli} could be found as commensal strains, pathogenic strains, or environmental strains. The pathogenic strains could pertein to these categories (which are not exclusive): enteropathogenic (EPEC), enteroinvasive (EIEC), enterotoxigenic (ETEC), diffusely adherent (DAEC), adherent invasive (AIEC), shiga-toxin producing (STEC), enteroaggregative(EAEC), extraintestinal pathogenic (ExPEC). Resistances to antibiotics make even more difficult the process of categorization of \emph{E. coli}. 
In 2011 in Germany, an outbreak of Stx-EAEC was responsible of the death of some people. An efficient counter-measure was found by sequencing the genome of those bacteria. 

%shigella info
Shigella is \emph{E. coli} with shiga toxin. some genes are responsible of the aggresomething. 

Most of the genes are on plasmids, circular, addition to chromosome, and can be moved easily horizontally. Plasmids between different strains can be moved in enterobacteriacae, this doesn't happen normally in other families.
Some \emph{E. coli} strains are even capable of causing tumors in humans: for example, colibactin-positive \emph{\emph{E. coli}} can cause colon and rectal cancer, creating mutations by attacking the human genome which are responsable of the of the cancer onset.



\begin{figure}[h]
\caption{\emph{\emph{E. coli}} pathogenic groups}
\centering
\includegraphics[width=0.6\textwidth]{...}
\end{figure}

several antigens can be used by taxonomists to cathegorize \emph{E. coli} strains. In particular there are the O, H, K antigens, respectively related to the somatic, the flagella and the capsule. O antigens are 171, Ks are 80 and Hs are 56.

\subsection{PanPhlAn - strain detection and characterization}

Pangenome-based Phylogenomic Analysis (PanPhlAn) is a strain-level metagenomic profiling tool for identifying the gene composition and in-vivo transcriptional activity of individual strains in metagenomic samples. PanPhlAnâ€™s ability for strain-tracking and functional analysis of unknown pathogens makes it an efficient tool for culture-free infectious outbreak epidemiology and microbial population studies. \href{http://segatalab.cibio.unitn.it/tools/panphlan/}{PanPhlAn reference}

% to be continued
