\graphicspath{{chapters/images/04/}}
\chapter{Sequencing data}
After all, genetics/genomics studies a code of a digital information (4bases/2bits). We should try to be as hypothesis driven as possible and use the already available and processed data to guide new data analysis. Be aware that, in genomics, data generation is the starting point of the study (the ratio of wet experiments vs computational effort is ~1:10)!\\
Given the biological problem at hand, we need to choose the optimal sequencing machine. To achieve this, we need to consider:
\begin{itemize}
\item Throughput;
\item Cost;
\item Read lenghts, 
\item Data output (reads per run);
\item Coverage; 
\item Seuencing errors (indel, substitution, CG delation, AT bias). Altough the error rate is decreasing wit new technologies;
\item Library preparation compatibility;
\item Speed (run time)
\end{itemize}
Suppose that we want to sequence a genome of a bacterium: which is the best machine that we can use?

\begin{description}
\item[Illumina NovaSeq]: if one wants to sequence a lot of DNA molecules at the same time, genomes, metagenomes. It canâ€™t go over the 300 bp readlines runned, but it has the highest throughput so far (3TB of output). It is capable of multiplexing, so we have a unique barcode for any input sample. 
\item[Illumina iSeq]: If you need to sequence shorter genomes. From iSeq to NextSeq (increasing the reads lengths). 
\item[NanoPore (minion)]: pocket-sized wet-lab free sequencer for DNA, RNA and (possibly) proteins, but the read lengths is smaller than Illumina's. The machine is cheap; the running flow is more expensive (going down by time). It's a real-time sequencer. 


\end{description}